packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -257,36 +211,79 @@ class DeepLinksController extends DisposableController
 
   final _androidAppLinks = <int, AppLinkSettings>{};
   final _iosLinks = <int, UniversalLinkSettings>{};
+  final _selectedAndroidVariantIndex = ValueNotifier<int>(0);
+  final _selectedIosConfigurationIndex = ValueNotifier<int>(0);
+  final _selectedIosTargetIndex = ValueNotifier<int>(0);
+
+  ValueListenable<int> get selectedAndroidVariantIndex =>
+      _selectedAndroidVariantIndex;
+  ValueListenable<int> get selectedIosConfigurationIndex =>
+      _selectedIosConfigurationIndex;
+  ValueListenable<int> get selectedIosTargetIndex => _selectedIosTargetIndex;
```

Comment:
nit: order private fields directly below their getter: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#order-other-class-members-in-a-way-that-makes-sense